Librarian/Series.js

```diff
@@ -36,21 +33,20 @@ Series.prototype.fetch = (done)=> {
   return;
 };
 
-
 /**
 SeriesコレクションからBookListコレクションを取得する
 **/
-Series.prototype.join = function(seriesItems, done){
-  var conditions = {
+Series.prototype.join = function join(seriesItems, done) {
+  const conditions = {
```

Comment:
__[jshint]__
*  'const' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).